Libs/MRML/Widgets/qMRMLUtils.cxx

```diff
@@ -182,3 +185,26 @@ void qMRMLUtils::qColorToColor(const QColor &qcolor, double* color)
     color[2] = qcolor.blueF();
     }
 }
+
+//------------------------------------------------------------------------------
+void qMRMLUtils::mimeDataToSubjectHierarchyItemIDs(const QMimeData* mimeData, vtkIdList* idList)
+{
+  if (!mimeData->hasFormat("text/uri-list") || !idList)
+    {
+    return;
+    }
+  idList->Reset();
+  foreach(QUrl url, mimeData->urls())
+    {
+    if (!url.isValid() || url.isEmpty())
+      {
+      continue;
+      }
+    if (url.scheme() != "mrml" || url.host() != "scene" || url.path() != "/subjecthierarchy/item")
```

Comment:
Do we have documentation for this scheme or should we consider this an implementation details ?

Reply:
This is implementation detail for now, as this URL is only applicable within the current scene of the current Slicer instance. 

The same approach could be used for opening real universal URLs directly in the viewer, but I don't have any meaningful use case in mind right now.